src/intrinsics.cpp

```diff
@@ -50,6 +50,14 @@ JL_DLLEXPORT uint32_t jl_get_LLVM_VERSION(void)
         ;
 }
 
+extern "C" JL_DLLEXPORT int8_t jl_is_memdbg() {
```

Comment:
How about spelling this out as `jl_is_memdebug` since remembering that these are spelled differently is going to be impossible? I know this isn't going to be heavily used, but still.